src/components/encoding-pane/property-editor-schema.ts

```diff
@@ -181,9 +352,42 @@ export function generateFormData(shelfId: ShelfId, fieldDef: ShelfFieldDef) {
       const prop = customProp.prop;
       const nestedProp = customProp.nestedProp;
       const propertyKey = nestedProp ? prop + '_' + nestedProp : prop;
-      formData[propertyKey] = fieldDef[prop] ? nestedProp ? fieldDef[prop][nestedProp] : fieldDef[prop] : undefined;
+      const fData = fieldDef[prop] ? nestedProp ? fieldDef[prop][nestedProp] : fieldDef[prop] : undefined;
+      // Display empty string when '?' is passed in to retrieve default value
+      // '?' is passed when formData is empty to avoid passing in empty string as a property/nestedProp value
+      formData[propertyKey] = fData === '?' ? '' : fData;
```

Comment:
Can you point me to a place where '?' is passed in?  This seems a bit brittle.

Reply:
https://github.com/vega/voyager/pull/799/files#diff-f09c2cf3fa6f34324d8cf3c87e68f796R72

This is used as a way to protect against users passing in blank data for text input form values. So if a user types in a domain, and then deletes all the values, it doesn't break Voyager.

Reply:
Thanks -- see https://github.com/vega/voyager/pull/799/files#r187163447